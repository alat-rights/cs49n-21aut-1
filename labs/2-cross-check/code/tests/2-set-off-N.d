2-set-off-N.o: 2-set-off-N.c test-helper.h ../rpi.h ../fake-pi.h
