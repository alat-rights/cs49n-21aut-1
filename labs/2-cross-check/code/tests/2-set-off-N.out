TRACE: calling pi code
TRACE: PUT32(0x20200028) = 0x1
TRACE: PUT32(0x20200028) = 0x2
TRACE: PUT32(0x20200028) = 0x4
TRACE: PUT32(0x20200028) = 0x8
TRACE: PUT32(0x20200028) = 0x10
TRACE: PUT32(0x20200028) = 0x20
TRACE: PUT32(0x20200028) = 0x40
TRACE: PUT32(0x20200028) = 0x80
TRACE: PUT32(0x20200028) = 0x100
TRACE: PUT32(0x20200028) = 0x200
TRACE: PUT32(0x20200028) = 0x400
TRACE: PUT32(0x20200028) = 0x800
TRACE: PUT32(0x20200028) = 0x1000
TRACE: PUT32(0x20200028) = 0x2000
TRACE: PUT32(0x20200028) = 0x4000
TRACE: PUT32(0x20200028) = 0x8000
TRACE: PUT32(0x20200028) = 0x10000
TRACE: PUT32(0x20200028) = 0x20000
TRACE: PUT32(0x20200028) = 0x40000
TRACE: PUT32(0x20200028) = 0x80000
TRACE: PUT32(0x20200028) = 0x100000
TRACE: PUT32(0x20200028) = 0x200000
TRACE: PUT32(0x20200028) = 0x400000
TRACE: PUT32(0x20200028) = 0x800000
TRACE: PUT32(0x20200028) = 0x1000000
TRACE: PUT32(0x20200028) = 0x2000000
TRACE: PUT32(0x20200028) = 0x4000000
TRACE: PUT32(0x20200028) = 0x8000000
TRACE: PUT32(0x20200028) = 0x10000000
TRACE: PUT32(0x20200028) = 0x20000000
TRACE: PUT32(0x20200028) = 0x40000000
TRACE: PUT32(0x20200028) = 0x80000000
TRACE: pi exited cleanly: 135 calls to random
